Assets/PatchKit Patcher/Scripts/AppData/Local/LocalMetaData.cs

```diff
@@ -40,6 +40,12 @@ private struct Data
             [DefaultValue("none")]
             [JsonProperty("product_key_encryption", DefaultValueHandling = DefaultValueHandling.Populate)]
             public string ProductKeyEncryption;
+            
+            [JsonProperty("main_executable")]
+            public string MainExecutable;
+            
+            [JsonProperty("main_executable_args")]
```

Comment:
Should be `start_exe_args` to make it compatible with Electron launcher.

Reply:
done